index.html

```diff
@@ -1,14 +1,17 @@
 <html>
-  <body>
-    <p id='server-time'></p>
+  <head>
     <script src="/socket.io/socket.io.js"></script>
     <script>
       var socket = io();
-      var el = document.getElementById('server-time');
+      var el;
```

Comment:
This doesn't need to be declared outside of the closure if you're query-ing it every time

Reply:
The element doesn't exist until after the event handler runs... so the option is to either declare it and then assign in the event handler or put the script at the bottom of the HTML